jaxopt/_src/bfgs.py

```diff
@@ -100,8 +100,14 @@ class BFGS(base.IterativeSolver):
       backtracking line search (default: 0.8).
     increase_factor: factor by which to increase the stepsize during line search
       (default: 1.5).
-    max_stepsize: upper bound on stepsize.
-    min_stepsize: lower bound on stepsize.
+    max_stepsize: upper bound on stepsize guess at start of each linesearch run
+      for linesearch_init='increase'.
+      Note that the linesearch is allowed to take a larger stepsize to satisfy
```

Comment:
It's a bit strange to say that. Maybe we need to make the distinction between `init_stepsize` and `max_stepsize` then? Also this remark is only for zoom ?

Reply:
- Yes, I can add a max_init_stepsize and keep the max_stepsize as it is. Max_stepsize would really enforce a maximum stepsize as expected. The min_stepsize does not enforce a minimum stepsize either during the search. So we may do the same behavior (add a min_init_stepsize and let a min_stepsize intervenes in the linesearches, this would let the users have the possibility to enforce the linesearches to jump if the minimum stepsize is reached and no stepsize has been found).
- Yes, only zoom may increase the initial stepsize during its search (Backtracking always reduces and HagerZhang ignores max_stepsize parameter).

Reply:
I kept the solvers as they were. So kept max_stepsize. I've simply changed the min_stepsize doc.